src/mds/topology/topology_stat.h

```diff
@@ -97,31 +97,29 @@ struct ChunkServerStat {
 };
 
 /**
- * @brief Topology统计模块，负责处理各种统计数据
+ * @brief Topology statistic module for managing its stats
  */
 class TopologyStat {
  public:
     TopologyStat() {}
     ~TopologyStat() {}
 
-    // 心跳来源
-
     /**
-     * @brief 更新心跳来源的chunkserver统计数据
+     * @brief Update the statistic of the chunkservers that sent heartbeat
      *
      * @param csId chunkserverId
-     * @param stat 心跳传来的chunkserver统计数据
+     * @param stat statistic brought by the heartbeat
      */
     virtual void UpdateChunkServerStat(ChunkServerIdType csId,
         const ChunkServerStat &stat) = 0;
     /**
-     * @brief 获取心跳来源的chunkserver统计数据
+     * @brief fetch statistic of the chunkserver that sent the heartbeat
```

Comment:
fetch the statistic information of  chunkservers that sent by heartbeat

Reply:
> fetch the statistic information of chunkservers that sent by heartbeat

done